src/pages/ReportDetailsPage.js

```diff
@@ -73,31 +67,39 @@ class ReportDetailsPage extends Component {
     constructor(props) {
         super(props);
 
-        this.notificationPreferencesOptions = {
-            default: {
-                value: CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS,
-                label: props.translate('reportDetailsPage.always'),
-
-            },
-            daily: {
-                value: CONST.REPORT.NOTIFICATION_PREFERENCE.DAILY,
-                label: props.translate('reportDetailsPage.daily'),
-            },
-            mute: {
-                value: CONST.REPORT.NOTIFICATION_PREFERENCE.MUTE,
-                label: props.translate('reportDetailsPage.mute'),
-            },
-        };
-
-        this.menuItems = ReportUtils.isArchivedRoom(this.props.report) ? []
-            : [
+        this.menuItems = [];
+        if (ReportUtils.isArchivedRoom(this.props.report)) {
+            return;
+        }
+
+        // All nonarchived chats should let you see their members
+        this.menuItems.push({
+            translationKey: 'common.members',
+            icon: Expensicons.Users,
+            subtitle: props.report.participants.length,
+            action: () => { Navigation.navigate(ROUTES.getReportParticipantsRoute(props.report.reportID)); },
```

Comment:
You mentioned some bugs on our call recently, and I was just writing something else in this file. I could be wrong but I think you need `this.props....` here. (same on the subtitle key above and the action for `settings` below.